interface/patient_file/deleter.php

```diff
@@ -174,15 +174,12 @@ function delete_document($document) {
 ?>
 <html>
 <head>
-<?php html_header_show();?>
```

Comment:
oh, just noted this. we should not remove this since some folks use this as a hook in custom code.